src/interoperability.md

```diff
@@ -1 +1,19 @@
-Contents will be added as part of https://github.com/sora-xor/sora-docs/pull/44
+# Interoperability
+
+This article is about the HASHI bridge - a decentralized two-way bridge connecting SORA and other blockchains (such as Ethereum and Polkadot). HASHI means bridge in Japanese, and it is the plumbing that links many different chains, like pipes connecting the world's assets. The initial version of this bridge was a significant component of the plans for the beta launch of the SORA network in Q1 2021.
```

Comment:
```suggestion
This article is about the HASHI bridge - a decentralized two-way bridge connecting SORA and other blockchains (such as Ethereum and Kusama). HASHI means bridge in Japanese, and it is the plumbing that links many different chains, like pipes connecting the world's assets. The initial version of this bridge was a significant component of the plans for the beta launch of the SORA network in Q1 2021.
```
We don't yet connect to Polkadot (soon hopefully though).

Reply:
https://github.com/sora-xor/sora-docs/pull/44/commits/b39c00a7d1c043d9624b6d1c67a9767f02a16f00